charts/datree-admission-webhook/templates/cluster-scanner.yaml

```diff
@@ -0,0 +1,146 @@
+metadata:
+  name: cluster-scanner-role
+  labels: {{ include "datree.labels" . | nindent 4 }}
+  {{- with .Values.customAnnotations }}
+  annotations: {{ toYaml . | nindent 4 }}
+  {{- end }}
+rules:
+  - apiGroups:
+      - "*"
+    resources:
+      - "*"
+    verbs:
+      - "get"
+      - "list"
+
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: cluster-scanner-rolebinding
+  labels: {{ include "datree.labels" . | nindent 4 }}
+  {{- with .Values.customAnnotations }}
+  annotations: {{ toYaml . | nindent 4 }}
+  {{- end }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-scanner-role
+subjects:
+  - kind: ServiceAccount
+    name: cluster-scanner-service-account
+    namespace: {{template "datree.namespace" .}}
+
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: datree-cluster-scanner-server
+  namespace: {{ template "datree.namespace" . }}
+  labels: {{ include "datree.labels" . | nindent 4 }}
+    owner: datree
+    app: "datree-cluster-scanner-server"
+  {{- with .Values.customAnnotations }}
+  annotations: {{ toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels: 
+      app: "datree-cluster-scanner-server"
+  template:
+    metadata:
+      labels: {{ include "datree.labels" . | nindent 8 }}
+        app: "datree-cluster-scanner-server"
+      {{- with .Values.customAnnotations }}
+      annotations: {{ toYaml . | nindent 8 }}
+      {{- end }}
+    spec:
+      {{- if .Values.nodeSelector }}
+      nodeSelector: {{- toYaml .Values.nodeSelector | nindent 8 }}
+      {{- end }}
+      {{- with .Values.affinity }}
+      affinity: {{ toYaml . | nindent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations: {{- toYaml .Values.tolerations | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{.Values.clusterScanner.rbac.serviceAccount.name}}
+      containers:
+        - name: cluster-scanner-server
+          env:
+            - name: CLUSTER_NAME
+              value: {{.Values.datree.clusterName}}
+            - name: DATREE_TOKEN
+            {{- if and .Values.datree.existingSecret (ne .Values.datree.existingSecret.name "") (ne .Values.datree.existingSecret.key "") }}
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.datree.existingSecret.name }}
+                  key: {{ .Values.datree.existingSecret.key }}
+            {{- else }}
+              value: "{{ .Values.datree.token }}"
+            {{- end }}
+            - name: DATREE_POLICY
+              value: {{.Values.datree.policy | default "Starter"}}
+            - name: DATREE_NAMESPACE
+              value: {{template "datree.namespace" .}}
+            - name: SCAN_INTERVAL_HOURS
+              value: "{{ now | dateModify "-5m" | date "04" }} */{{ .Values.datree.scanIntervalHours | default 1 }} * * *" # every hour, starting at least 55 minutes after helm installation
```

Comment:
I think this should be just the number as-is.
The cron template was just a workaround.